docs/getting-started.md

```diff
@@ -4,6 +4,10 @@ sidebar_position: 1.1
 
 # Getting Started
 
+:::caution
```

Comment:
Did we not remove this?

Reply:
idk, it looks like it is live:
![image](https://user-images.githubusercontent.com/477945/152164138-90372de3-38ee-4be6-b7ae-c9a07436b6e9.png)